views/reducers/session.jsx

```diff
@@ -15,13 +16,22 @@ const user = function user(state = {
         isProcessing: false,
         signedIn: true,
         email: action.email,
+        authError: false,
       });
     case SIGNED_OUT:
       return Object.assign({}, state, {
         isProcessing: false,
         signedIn: false,
         email: '',
       });
+    case AUTH_ERROR:
+      return Object.assign({}, state, {
+        authError: true,
```

Comment:
at this point, we are done processing the user so we should set isProcessing to false